test/TestProjects/Lro-Basic-TypeSpec/src/Generated/LroBasicTypeSpecClient.cs

```diff
@@ -70,7 +73,10 @@ public virtual async Task<Operation> CreateProjectAsync(WaitUntil waitUntil, Pro
             return await CreateProjectAsync(waitUntil, resource.ToRequestContent(), context).ConfigureAwait(false);
         }
 
-        /// <summary> The most basic operation. </summary>
+        /// <summary>
+        /// DEPRECATED: Use LongRunningResourceCreateWithServiceProvidedName from the ResourceOperations interface.
```

Comment:
why are those deprecated information for typespec shown in generated SDK?
I didn't see the typespec change, so is this a problem for our emitter or typespec compiler?

Reply:
as above